msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -93,39 +224,168 @@ public void onTaskCompleted(List<ICacheRecord> cacheRecords) {
                     @Override
                     public void onError(Exception exception) {
                         callback.onError(exception);
+
                     }
                 });
     }
 
+    private void checkCurrentAccountNotifyCallback(final CurrentAccountCallback callback, List<ICacheRecord> newAccountRecords){
```

Comment:
Either replace this with the code in getCurrentAccountFromSharedDevice's callback, or call this function over there.

Reply:
Agreed.  Fixed.